compiler/rustc_next_trait_solver/src/solve/normalizes_to/anon_const.rs

```diff
@@ -21,6 +21,13 @@ where
             self.instantiate_normalizes_to_term(goal, normalized_const.into());
             self.evaluate_added_goals_and_make_canonical_response(Certainty::Yes)
         } else {
+            // FIXME(BoxyUwU) FIXME(min_generic_const_args): I could not figure out how to write a test for this
+            // as we don't currently support constants in the type system with impossible predicates. It may become
+            // possible once `min_generic_const_args` has progressed more.
```

Comment:
Maybe this is possible with GCE? I don't really want to write a load bearing GCE test lol